cmake/v_library.cmake

```diff
@@ -53,7 +53,7 @@ set(V_COMMON_INCLUDE_DIRS
 # TODO: Implement "ALWAYSLINK"
 function(v_cc_library)
   cmake_parse_arguments(V_CC_LIB
-    "DISABLE_INSTALL;PUBLIC"
+    "DISABLE_INSTALL;PUBLIC;OBJECT"
```

Comment:
Do we need to update the comment around line 20? I'm fine with doing this as an add-on patch at the end.

Reply:
oh yeh ill update the docs. thanks